test/data/test_functional.py

```diff
@@ -94,14 +94,14 @@ def test_BasicEnglishNormalize(self):
         basic_eng_norm = basic_english_normalize()
         experimental_eager_tokens = basic_eng_norm(test_sample)
 
-        jit_basic_eng_norm = torch.jit.script(basic_eng_norm.to_ivalue())
+        jit_basic_eng_norm = torch.jit.script(basic_eng_norm)
         experimental_jit_tokens = jit_basic_eng_norm(test_sample)
 
         basic_english_tokenizer = data.get_tokenizer("basic_english")
         eager_tokens = basic_english_tokenizer(test_sample)
 
         assert not basic_eng_norm.is_jitable
-        assert basic_eng_norm.to_ivalue().is_jitable
+        assert basic_eng_norm.__prepare_scriptable__().is_jitable
```

Comment:
I'd interact with this via torch.jit.script. These methods are not meant to be called directly.

Reply:
Here we want to check if the building block backed by torchbind is jitable. Although we don't recommend the torchbind version for eager mode, I think we still need a test to cover it. By calling `torch.jit.script`, we are testing the jit mode, which is not the same thing with the torchbind building block.

Same thing for the pickle support.